src/main/java/com/eva/commons/util/DateUtil.java

```diff
@@ -74,8 +75,19 @@ public static String dateToString(LocalDate date) {
      * @param end cannot be empty.
      */
     public static int getDaysBetween(LocalDate start, LocalDate end) {
-        requireNonNull(start);
-        requireNonNull(end);
+        requireAllNonNull(start, end);
         return Period.between(start, end).getDays() + 1;
     }
+
+    /**
+     * Returns a list of LocalDates between the two given {@code dates}.
```

Comment:
Minor, Good to specify if the list returned is inclusive or exclusive of the start and end date